AssemblyInfo.cs

```diff
@@ -0,0 +1,6 @@
+﻿using System.Runtime.CompilerServices;
+
+// This is necessary to allow cross-communication between assemblies for classes in the UnityEditor.Experimental.EditorVR
+// namespace; The plan is that all code will eventually be within a single assembly.
+
+[assembly: InternalsVisibleTo("EXR")]
```

Comment:
Is this still the name of our Assembly Definition file? I assume so, since it is working.

Reply:
Yeah, it is. Since you mention it, we should probably change it to `EditorXR`. There are still a few remaining places where we call things `EditorVR`. I have a task out for cleaning up the remaining naming issues, and I think changing the assembly name to `EditorXR` makes sense to do then.